webapp/src/Migrations/Version20200118092658.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20200118092658 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return 'rename team.externalid to team.icpcid';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('DROP INDEX externalid ON team');
+        $this->addSql('ALTER TABLE team CHANGE externalid icpcid VARCHAR(255) DEFAULT NULL COLLATE utf8mb4_bin COMMENT \'Team ID in the ICPC system\'');
+        $this->addSql('CREATE UNIQUE INDEX icpcid ON team (icpcid(190))');
```

Comment:
out of curiosity: what does the 190 mean here?

Reply:
Limit the index to the first 190 characters, which is needed for not-so-new MySQL's, since indices on varchars can not exceed 767 bytes, and utf8mb4 uses 4 bytes, so 190*4 < 767